src/common/transformations/src/transformations/common_optimizations/augru_cell_fusion.cpp

```diff
@@ -81,7 +81,11 @@ ov::pass::AUGRUCellFusion::AUGRUCellFusion() {
     // only Sigmoid is supported in the current version of AUGRUCell
     auto sigmoid = wrap_type<ov::op::v0::Sigmoid>({add_1});
     auto split = wrap_type<ov::op::v1::Split>({sigmoid, any_input()});
-    auto multiply = wrap_type<ov::op::v1::Multiply>({split, any_input()});
```

Comment:
Is still any test with use the `any_input`?
It should match for any output on split